src/auto/functions.rs

```diff
@@ -413,9 +410,12 @@ pub fn set_show_events(show_events: bool) {
     }
 }
 
-//pub fn setting_get(name: &str, value: /*Ignored*/&mut glib::Value) -> bool {
-//    unsafe { TODO: call ffi::gdk_setting_get() }
-//}
+pub fn setting_get(name: &str, value: &mut glib::Value) -> bool {
```

Comment:
Does the `GValue` have to be initialized with the correct type before calling? Or all zeroes?

You probably want to ensure either inside the function and return a `Option<Value>`

Reply:
Oh true! I went too fast with this one...